src/content/en/tools/workbox/guides/_toc.yaml

```diff
@@ -6,7 +6,9 @@ toc:
   path: /web/tools/workbox/guides/precache-files
 - title: "Route Requests"
   path: /web/tools/workbox/guides/route-requests
-- title: "Configure Workbox"
+- title: "Configure the Build Tools"
+  path: /web/tools/workbox/guides/configure-build-tools
+- title: "Configure the Runtime Options"
```

Comment:
Could we move the above configuration in the module pages where relevant and then keep the guides IA as is and then link up the appropriate config info to each module.